SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2423,6 +2423,18 @@ void USpatialReceiver::ResolveIncomingOperations(UObject* Object, const FUnrealO
 			}
 		}
 
+		ObjectsToInspect.Add(ChannelObjectsToBeResolved{ DependentChannel, ReplicatingObject, RepState });
+	}
+
+	for (const auto& Objects : ObjectsToInspect)
+	{
+		USpatialActorChannel* DependentChannel = Objects.Channel;
+		if (!Objects.Object.IsValid())
+		{
+			continue;
+		}
+		UObject* ReplicatingObject = Objects.Object.Get();
+		FSpatialObjectRepState* RepState = Objects.RepState;
```

Comment:
Does Objects.Object == nullptr mean that Repstate is valid?

Reply:
I think so. There is still the question of the owning ActorChannel being closed.....
The only thing I could think of to maybe check for that is `(Channel.Actor == ReplicatingObject->GetTypedOuter<AActor>())` but I am not fully confident that is sufficient or even correct.